test/TestProjects/Models-Cadl/Generated/cadl.json

```diff
@@ -649,7 +649,7 @@
      "Description": "",
      "Type": {
       "$id": "82",
-      "Name": "TimeSpan",
+      "Name": "duration",
```

Comment:
why is this change?

Reply:
There are some other changes like:
Datetime=>zonedDateTime,
bool=>boolean
Int32=>int32

Are they expected?

Reply:
This is exactly what Crystal and I discussed above: https://github.com/Azure/autorest.csharp/pull/2747#discussion_r984156640

If a property is of type `ResourceLocation` which is actually a `string`. Previously we used `string` as the value for both `Kind` and `Name`. I think we should use `string` as `Kind`, but keep the original name for `Name`.